### DISCUSSION TITLE
Welcome to saj-mqtt-ha Discussions!

### DISCUSSION THREAD

**Discussion by paolosabatino** [paolosabatino/saj-mqtt-ha#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by walkjivefly** [comment]
Hurrah! I take a look at the Telegram group occasionally, but not being a native, or even a competent Spanish speaker, I find it too hard to engage with.

I (had) installed a solar PV system in July this year, and installed HomeAssistant on a RPi to try to exercise a bit of control over when/how the (excess) generated power is used. I have the impression that SAJ inverters are not the easiest to use with HomeAssistant, but that's what I've got. After an abortive initial attempt at using the mqtt integration I'm making do with one of the SAJ cloud integrations.
